src/js/app.js

```diff
@@ -0,0 +1,95 @@
+var picArray = [];
+var count = 0;
+
+
+$(document).ready(function(){
+
+  $('#textInput').focus();
+
+	$('#formId').on('submit', function(e){
+
+		e.preventDefault();
+
+
+
+    word = $('#textInput').val();
```

Comment:
Keep in mind that `word` wasn't declared with `var`. If you do this, `word` is "hoisted" to global scope.